### PR TITLE
Upload markdown

### PR DESCRIPTION
**Pull request by M8is**
Pull request for #48

### CODE REVIEW

File: mubench.pipeline/tasks/implementations/publish_findings_task.py [hunk 1]
```diff
@@ -62,14 +62,17 @@ def process_project_version(self, project: Project, version: ProjectVersion) ->
 
         logger.info("Extracting target source code...")
```
Comment (salsolatragus) [inline mubench.pipeline/tasks/implementations/publish_findings_task.py]: This message is no longer true.

File: mubench.pipeline/tests/data/test_finding.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 from typing import Dict
 from unittest.mock import patch
 
-from nose.tools import assert_equals
+from nose.tools import assert_equals, assert_raises
```
Comment (salsolatragus) [inline mubench.pipeline/tests/data/test_finding.py]: Import needed?

**Comment by joschli** [comment]
When i try to publish findings, it crashes with an UnsupportedTypeError because it is trying to convert an Integer to markdown.

**Comment by salsolatragus** [comment]
@M8is could you please have a look at the problem that @joschli reported above? @joschli can you maybe provide the error trace to help with debugging?

**Comment by joschli** [comment]
> Traceback (most recent call last):
  File "./mubench.pipeline/benchmark.py", line 166, in <module>
    benchmark.run()
  File "./mubench.pipeline/benchmark.py", line 136, in run
    self.runner.run()
  File "/mubench/mubench.pipeline/task_runner.py", line 33, in run
    response = task.process_project(project)
  File "/mubench/mubench.pipeline/tasks/project_version_task.py", line 17, in process_project
    response = self.process_project_version(project, version)
  File "/mubench/mubench.pipeline/tasks/implementations/publish_findings_task.py", line 70, in process_project_version
    postable_data = self._prepare_post(potential_hit, version_compile)
  File "/mubench/mubench.pipeline/tasks/implementations/publish_findings_task.py", line 101, in _prepare_post
    markdown_dict = self._to_markdown_dict(finding)
  File "/mubench/mubench.pipeline/tasks/implementations/publish_findings_task.py", line 130, in _to_markdown_dict
    markdown_dict[key] = as_markdown(value)
  File "/mubench/mubench.pipeline/utils/web_util.py", line 109, in as_markdown
    raise UnsupportedTypeError(value)
utils.web_util.UnsupportedTypeError: Unsupported type <class 'int'> of 2

This is the traceback for the call: ./mubench publish findings GrouMiner 1 --only aclang

**Comment by M8is** [comment]
Fixed. This is supposed to happen for unsupported types and may happen again in the future if other types are added to the upload data. Supported types can be extended by a respective case in `as_markdown`.

**Comment by salsolatragus** [comment]
Agreed. We should make sure we support all types we currently use, though ;)

Can you rebase the branch? I'm not sure what's causing the conflict. Tell me if you need help.

**Comment by salsolatragus** [comment]
Changes taken over to #73.

**Pull request closed without merging**